social/communication_and_social_stuff.md

```diff
@@ -0,0 +1,48 @@
+# Communication and Social Stuff
+## Email Lists/ Google Groups
+- saturn-dev
+- dsp-analysis-platform
+
+## Calendars
+- DSDE Events
+
+## Slack Channels
```

Comment:
@Shakespeared  has a slack channels section in her current PR so maybe some de-deduping could be worked out here?

Reply:
Seconded that we should merge the two slack channel sections. I was also brainstorming with @ehanna4 about a way to organize the channel suggestions. My current idea is 'Engineering/Technical,' 'Social/Fun,' and 'Team/Work.'

Reply:
Ohhh snap I didn't see that!
I love that
Where do you folks think would be a good choice to put them?
Also, I definitely think we should just do fun as examples (since there's so many) except for the DSP based ones since you can miss those

Reply:
@cameron-ardell no worries haha! We could also give some 'fun' examples then link out to the complete list of channels (which I think exists somewhere though I don't remember where) 

TOL: What does everyone think about having a third category for social/communication stuff, since it doesn't quite fit under 'tooling' or 'identity'? Then we could throw the slack channels suggestions and social stuff in there. (Although there is an argument for that going in 'identity' as well so I'm not sure.)

Reply:
I would be a fan of making a new category. I think the more buckets, the better. (As long as they are sensible buckets)

Reply:
I made a new bucket for this called "communication and social" in this PR